File: Examples/ProgrammingWithDocuments/WorkingWithBookmarks/WorkingWithBookmarksExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammingWithDocuments.WorkingWithBookmarks
	<TestClass, TestFixture> _
	Public Class WorkingWithBookmarksExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub CopyBookmarkedText()
	            TestHelper.SetDataDir("ProgrammingWithDocuments/WorkingWithBookmarks/CopyBookmarkedText")
	
	            CopyBookmarkedTextExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub UntangleRowBookmarks()
	            TestHelper.SetDataDir("ProgrammingWithDocuments/WorkingWithBookmarks/UntangleRowBookmarks")
	
	            UntangleRowBookmarksExample.Program.Main()
        End Sub

	End Class
End Namespace

File: Examples/ProgrammingWithDocuments/WorkingWithBookmarks/WorkingWithBookmarksExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammingWithDocuments.WorkingWithBookmarks
{
    [TestClass, TestFixture]
    public class WorkingWithBookmarksExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void CopyBookmarkedText()
        {
            TestHelper.SetDataDir("ProgrammingWithDocuments/WorkingWithBookmarks/CopyBookmarkedText");

            CopyBookmarkedTextExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void UntangleRowBookmarks()
        {
            TestHelper.SetDataDir("ProgrammingWithDocuments/WorkingWithBookmarks/UntangleRowBookmarks");

            UntangleRowBookmarksExample.Program.Main();
        }

    }
}
